Scripts/script-Malwr.yml

```diff
@@ -1,8 +1,8 @@
 versionedfields: {}
```

Comment:
do we want both script AND integration of malwr?
I think the integration is enough